webanno/webanno
# #1420 - Hide layer in Annotation UI if only one layer is available

**What's in the PR**
- hide layer selection and annotation layer from AnnotationDetailEditorPanel if only one span layer is available in a project

**How to test manually**
1. Make a new project
2. Have only one span layer
3. Open the new project
4. Layer selection should be not visible
5. Create an annotation
6. Layer of annotation should be not visible
7. Open old project with multiple span layers
8. Layer selection and layer of annotation should still be visible

**Automatic testing**

**Documentation**
